<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistsWindow</class>
 <widget class="QWidget" name="PlaylistsWindow">
  <property name="windowTitle">
   <string>–ü–ª–µ–π–ª–∏—Å—Ç—ã</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">

   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>–ü–ª–µ–π–ª–∏—Å—Ç—ã</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 18pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>

   <!-- üîΩ –ö–Ω–æ–ø–∫–∞ –¥–ª—è —Å–æ–∑–¥–∞–Ω–∏—è –ø–ª–µ–π–ª–∏—Å—Ç–∞ -->
   <item>
    <widget class="QPushButton" name="btnAddPlaylist">
     <property name="text">
      <string>‚ûï –°–æ–∑–¥–∞—Ç—å –ø–ª–µ–π–ª–∏—Å—Ç</string>
     </property>
    </widget>
   </item>

   <!-- üîΩ –°–∫—Ä–æ–ª–ª –¥–ª—è –æ—Ç–æ–±—Ä–∞–∂–µ–Ω–∏—è –≤—Å–µ—Ö –ø–ª–µ–π–ª–∏—Å—Ç–æ–≤ -->
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QVBoxLayout" name="playlistLayout"/>
     </widget>
    </widget>
   </item>

   <!-- üîΩ –ö–Ω–æ–ø–∫–∏ —É–ø—Ä–∞–≤–ª–µ–Ω–∏—è -->
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="text">
        <string>‚úèÔ∏è –†–µ–¥–∞–∫—Ç–∏—Ä–æ–≤–∞—Ç—å</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>üóë –£–¥–∞–ª–∏—Ç—å</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
